config/locales/modules/alpha.yml

```diff
@@ -124,16 +124,93 @@ en:
       1-2-1:
         heading: 1-2-1
         body: First page in second submodule
-      1-2-2:
-        heading: 1-2-2
+      1-2-1-1:
+        heading: 1-2-1-1
         body: |
           %Warning: people like stuff!%
-      1-2-3:
+      1-2-1-2:
         heading: 1-2-3
```

Comment:
heading and body content needs to change

Reply:
Heading updated.